FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -111,6 +111,17 @@
  */
 @property (readwrite, nonatomic, assign) FBSnapshotTestCaseAgnosticOption agnosticOptions;
 
+/**
+ When set, resets the test name to the specified one, which in turn also resets the folder in
+ which the snapshopt is going to be saved.
+ */
+@property (readwrite, nonatomic, assign) NSString *testName;
+
+/**
+ If set to true, this will remove the `test` from the image name.
+ */
+@property (readwrite, nonatomic, assign) BOOL trimTestFromImageName;
```

Comment:
What's the relation of this property to the other one you've added? If you set `testName` to `testThing` and also set `trimTestFromImageName` to `YES` will `testThing` -> `Thing`?